tests/5.0/target/test_target_parallel_default.c

```diff
@@ -0,0 +1,42 @@
+//===------ test_target_default.c --------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// Tests the target directive with parallel for + default clause.
+//
+//===------------------------------------------------------------------------===//
+
+#include <stdio.h>
+#include "omp.h"
+#include "ompvv.h"
+
+#define N 100
+
+
+int IfTstPassed = 1; // 1 is passed 0 is failed
+
+int main(int argc, char** argv) {
+  OMPVV_TEST_OFFLOADING;
+  int count = 123, errors = 0;
+#pragma omp target data map(tofrom: count)
+#pragma omp target parallel for shared(count, IfTstPassed) default(none)
+  for (int i = 0; i < N; ++i) {
+    if (count != 123) {
+#pragma omp atomic
+      IfTstPassed--;
+    }  
+  }
+
+  count = 123;
+#pragma omp target parallel for default(shared)
```

Comment:
Should we add `map(tofrom: IfTstPassed)` here too? Otherwise, `IfTstPassed` will become `firstrprivate` to the generated target task (even though `IfTstPassed` will be shared within the parallel region), and thus this test will not fail at all.